constitution.tex

```diff
@@ -520,9 +520,11 @@
 \asubsubsection{The Introductory Process}
 The Introductory Process is designed to provide an easy means for Introductory Members to meet existing House members, learn House history, demonstrate their involvement potential to House, and allow existing House members to evaluate them for Active Membership.
 \asubsubsubsection{The Evaluation Period}
-The Process will begin between the first and fourth week of each semester, then last between six (6) and ten (10) weeks, starting when the Evaluations Director initiates it.
+The Process will occur either once or twice per semester, and will last for six (6) weeks.
+The Process shall be initiated by the Evaluations Director during either the first or second week of each semester.
 The Executive Board may hold a closed ballot Simple Majority vote to extend the Introductory Process or the Introductory Packet.
 If an Introductory Process is extended, the Introductory Evaluation shall occur at the termination of the extended Process, which may be outside of the period defined in \ref{Introductory Evaluation}.
+If deemed necessary by the Evaluations Director, or by an Executive Board simple majority vote, the second intro process must begin within nine (9) days of the first process ending.
```

Comment:
Some additional wording here to allow packets to start for the first two week of the second introductory process would be good